src/components/bcros/todo/items/Affiliation.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="flex flex-row" data-cy="todoItem-affiliation">
+    <div
+      class="p-1 min-h-12"
+      data-cy="todoItemTitle-affiliation"
+    >
+      <strong>{{ $t('title.todoItem.affiliationRequest') }}</strong><br>
+      <span>{{ $t('title.todoItem.from') }}: {{ todoItem.affiliationInvitationDetails?.fromOrgName }}</span>
+    </div>
+
+    <div
+      class="p-1"
+      data-cy="todoItemExpander-affiliation"
+    >
+      <UButton
+        variant="ghost"
+        leading-icon="i-mdi-information-outline"
+        class="p-3 w-40 h-10"
+        data-cy="todoItem-affiliation-showMore"
+        @click="isExpanded=!isExpanded"
+      >
+        {{ isExpanded ? $t('button.todoItem.hideDetails') : $t('button.todoItem.showDetails') }}
+      </UButton>
+    </div>
+
+    <div class="flex flex-row gap-1 ml-auto p-1" data-cy="todoItemActions">
+      <UButton
+        variant="outline"
+        class="px-3 w-40 h-10"
+        data-cy="todoItem-affiliation-doNotAuthorizeButton"
+        @click="todosStore.authorize(todoItem.affiliationInvitationDetails.id, false)"
+      >
+        <span class="w-full text-center"> {{ $t('button.todoItem.doNotAuthorize') }}</span>
+      </UButton>
+      <UButton
+        class="px-3 w-40 h-10"
+        data-cy="todoItem-affiliation-authorizeButton"
+        @click="todosStore.authorize(todoItem.affiliationInvitationDetails.id, true)"
+      >
+        <span class="w-full text-center"> {{ $t('button.todoItem.authorize') }}</span>
+      </UButton>
+    </div>
+  </div>
+  <div
+    v-if="isExpanded"
+    data-cy="todoItemBody-affiliation"
```

Comment:
Pending on whether we do accordian or not I guess, but it should have a transition class for hiding / expanding (adding something like `transition-all` here may work or we might need to add it on a div that wraps this so that it transitions both hiding and showing)